Add `Project Prism 2024: Registrations Now Open!` news post (#11273)
ppy/osu-wiki: news/2024/2024-04-19-project-prism-2024-registrations-now-open.md

---
layout: post
title: "Project Prism 2024: Registrations Now Open!"
date: 2024-04-19 15:00:00 +0000
---

Welcome to Project Prism, a 1v1 open-ranked osu!taiko tournament with a unique Raid Battle mechanic!

![](/wiki/shared/news/2024-04-19-project-prism-2024-registrations-now-open/banner.jpg)

> As the successor to the [osu!taiko Malaysian Tournament (OTMT)](/wiki/Tournaments/OTMT/2022) and [Global Taiko Malaysian Showdown (GTMS)](https://gtsosu.com/gtms/home) series, [Project Prism](https://prism.osumalaysia.org/) bursts through the gates endeavouring to deliever a brand-new osu! tournament experience, featuring a full storyline, SEVEN (7) original characters, music releases, artworks, and much more!

Reviewer: ```suggestion
As the successor to the [osu!taiko Malaysian Tournament (OTMT)](/wiki/Tournaments/OTMT/2022) and [Global Taiko Malaysian Showdown (GTMS)](https://gtsosu.com/gtms/home) series, [Project Prism](https://prism.osumalaysia.org/) bursts through the gates endeavouring to deliever a brand-new osu! tournament experience, featuring a full storyline, **seven** (7) original characters, music releases, artworks, and much more!
```
Perhaps bold this instead of capitalising. Could confuse people since other tournament abbreviations are used in the same sentence, but this one is not a tournament series.

> As the successor to the [osu!taiko Malaysian Tournament (OTMT)](/wiki/Tournaments/OTMT/2022) and [Global Taiko Malaysian Showdown (GTMS)](https://gtsosu.com/gtms/home) series, [Project Prism](https://prism.osumalaysia.org/) bursts through the gates endeavouring to deliever a brand-new osu! tournament experience, featuring a full storyline, **seven** (7) original characters, music releases, artworks, and much more!

Reviewer 2: ```suggestion
As the successor to the [osu!taiko Malaysian Tournament (OTMT)](/wiki/Tournaments/OTMT/2022) and [Global Taiko Malaysian Showdown (GTMS)](https://gtsosu.com/gtms/home) series, [Project Prism](https://prism.osumalaysia.org/) bursts through the gates endeavouring to deliver a brand-new osu! tournament experience, featuring a full storyline, **seven** (7) original characters, music releases, artworks, and much more!
```

Each boss character is associated with an original tiebreaker song release each week starting from the Round of 32. An introductory round is available before then (or in other words *right now*), featuring osu!taiko Malaysia's poster girl IRIS.

![](/wiki/Tournaments/Project_Prism/2024/img/character-cards.png)

Worried about the beatmaps being too difficult? Not a problem! All boss character beatmaps will feature a full difficulty mapset, allowing players of all levels to join in the fun!

To make things more interesting, the Raid Battle mechanic also comes with an added character card system, allowing you to equip up to 5 cards for each score submission to multiply your damage against the boss.

Character cards can be obtained via a *gacha* system, meaning players can earn tickets from weekly missions in order to redeem random cards.

And here's the kicker:

**Everyone is eligible to participate, even non-tournament players!**

## How do I register for the tournament?

**[Click here!](https://prism.osumalaysia.org/registration)**

> Tournament participation is available for all osu!taiko players, with the exception of those that have **violated the osu! Community rules within the past 12 months.**

Reviewer 2: ```suggestion
Tournament participation is available for all osu!taiko players, except those that have **violated the osu! Community rules within the past 12 months.**
```

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/N6PRgV6Hu5s" frameborder="0" allowfullscreen></iframe>
</div>

## Tournament schedule

| Event | Dates |
| --: | :-- |
| Registration phase | April 13–April 27 (23:59 UTC) |
| Qualifier showcase | May 5 |
| Qualifier stage | May 11 |
| Round of 64 | May 18–May 19 |
| Round of 32 | May 25–May 26 |
| Round of 16 | June 1–June 2 |
| Quarterfinals | June 8–June 9 |
| Semifinals | June 15–June 16 |
| Finals | June 22–June 23 |
| Grand Finals | June 29–June 30 |

## Prizes

- **1st:** Physical trophy, merchandise, profile badge (pending)
- **2nd:** Physical trophy, merchandise
- **3rd:** Physical trophy, merchandise

## Raid Battles

![](/wiki/shared/news/2024-04-19-project-prism-2024-registrations-now-open/raid-banner.jpg)

Project Prism introduces a *Raid Battle* system, where players cooperate to defeat boss characters by submitting scores on Project Prism's tiebreaker maps!

Each boss character is associated with an original tiebreaker song release each week starting from the Round of 32.

![](/wiki/Tournaments/Project_Prism/2024/img/character-cards.png)

> An introductory round is available before then (read: *right now*), featuring osu!taiko Malaysia's poster girl IRIS. She seems to already have been defeated, so perhaps we should increase the HP... but you can still submit scores and complete *missions* for bosses all the way until the tournament ends.

Reviewer 2: maybe?
```suggestion
An introductory round is available before then (read: *right now*), featuring osu!taiko Malaysia's poster girl, IRIS. She seems to already have been defeated, so perhaps we should increase the HP... but you can still submit scores and complete *missions* for bosses all the way until the tournament ends.
```

Worried about the beatmaps being too difficult? Not a problem! All boss maps include a full difficulty spread, allowing players of all levels to join in the fun!

To make things more interesting, the Raid Battle mechanic also comes with an added character card system, allowing you to equip up to 5 cards for each score submission to multiply your damage against the boss.

Character cards can be obtained via a *gacha* system, meaning players can earn tickets from weekly missions (we mentioned those, right?) in order to redeem random cards.

And here's the kicker:

**Everyone is eligible to participate, even non-tournament players!**

[View more details on the website](https://prism.osumalaysia.org/raid-battle/info) or [jump right in](https://prism.osumalaysia.org/raid-battle/1)!

## How do I register for the tournament?

**[Click here!](https://prism.osumalaysia.org/registration)**

Tournament participation is available for all osu!taiko players, except those that have **violated the osu! Community rules within the past 12 months.**

> Joining the official [osu!taiko Malaysia Discord Server](https://discord.com/invite/osutaikomy) is mandatory for all tournament participants, but we also highly encourage everyone else to join in as well for frequent updates on the event, including the Raid Battles!

Reviewer 2: ```suggestion
Joining the official [osu!taiko Malaysia Discord server](https://discord.com/invite/osutaikomy) is mandatory for all tournament participants, but we also highly encourage everyone else to join in as well for frequent updates on the event, including the Raid Battles!
```
